Resources/apiresources.robot

```diff
@@ -855,3 +855,46 @@ fetchJsonRespContainingAHUPropertiesOfSpecificGroup
     ${query}=    gqlQueries.getAHUsPropertiesOfSpecificGroup  ${group_name}
     ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
     return from keyword    ${json_dictionary}
+
+    #=======added for Alarm testcases
+    #Gets the list of message for alaram AHUFailToTurOff
+queryToFetchAlarmMsgAHUFailToTurOff
+    ${query}=    gqlQueries.getAlarmMsgAHUFailToTurOff
+    ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
+    #fetching the Alarm  message for Ahu and putting in list
+    ${total}=    get length   ${json_dictionary['data']['alarms']}
+    @{alarm_message_list}=    Create List
+    FOR    ${i}    IN RANGE   ${total}
+        Append To List    ${alarm_message_list}    ${json_dictionary['data']['alarms'][${i}]['message']}
+    END
+    return from keyword    ${alarm_message_list}
+
+    #Gets the list of the Ahu-pathname(ex:"NoBindings / NB-AHU-10") that are in mistmatch
+queryToFetchAHUMismatchForAHUFailToTurOffAlarm
```

Comment:
getAllAHUsWithMismatchAHUState

Reply:
updated to getAllAHUInGroupInMismatchState